src/datepicker/datepicker.ts

```diff
@@ -158,7 +166,8 @@ export class NgbDatepickerContent {
       </ngb-datepicker-navigation>
     </div>
 
-    <div class="ngb-dp-content" [class.ngb-dp-months]="!contentTemplate" #content>
+    <div class="ngb-dp-content" [class.ngb-dp-months]="!contentTemplate" #content
+         role="grid" aria-labelledby="selected-date">
```

Comment:
The `aria-labelledby` here may have some issues.

1) I may just be missing it, but where is the corresponding element with id of `selected-date`?
2) Having a hard-coded ID isn't a great idea in any case - e.g. if there's multiple date pickers on the screen.
3) I'm not sure that the selected date is really an accurate label for the entire date picker.